packages/ui-expandable/src/Expandable/index.tsx

```diff
@@ -22,22 +22,25 @@
  * SOFTWARE.
  */
 import React, { Component } from 'react'
-import { uid } from '@instructure/uid'
-import { createChainedFunction } from '@instructure/ui-utils'
+import { createChainedFunction, hashInstance } from '@instructure/ui-utils'
 
 import { propTypes, allowedProps } from './props'
 import type { ExpandableProps, ExpandableState } from './props'
 
+import { withSSR } from '@instructure/ui-react-utils'
+
 const toggleExpanded = ({ expanded }: { expanded: boolean }) => ({
   expanded: !expanded
 })
 
+//@ts-expect-error check out this later
+@withSSR()
```

Comment:
These decorators usually come between the annotation and the class declaration, don't they?

Reply:
Yeah, right, fixed it